titus-server-master/src/main/java/com/netflix/titus/master/loadbalancer/service/DefaultLoadBalancerService.java

```diff
@@ -198,4 +212,60 @@ public void deactivate() {
     Observable<Batch<TargetStateBatchable, String>> events() {
         return engine.events();
     }
+
+    /**
+     * Temporary utility to backfill all targets from registered load balancers to the store, since targets were not
+     * being persisted before.
+     * <p>
+     * Once all current state has been backfilled (i.e.: this runs successfully at least once), this is not needed
```

Comment:
This is only correct, if the load balancer is managed by one cell?

Reply:
existing code requires that a load balancer is managed only by one cell (single titusmaster leader), so that is assumed. After backfill happens and this property is disabled, multiple cells can start managing the same load balancer.